cra-rxjs-styled-components/src/components/user-repos/UserRepos.tsx

```diff
@@ -0,0 +1,102 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { tap } from 'rxjs';
+import { ORG_REPO_LIST } from '../../constants/url.constants';
+import { fromFetchWithAuth } from '../../hooks/auth/from-fetch-with-auth';
+import { Star } from '@styled-icons/heroicons-outline';
+import { Repo } from './types';
+import {
+  Aside,
+  BadgeWrapper,
+  Containers,
+  Content,
+  Description,
+  Header,
+  HeadingLink,
+  RepoListWrapper,
+  StarBtn,
+  StarIcon,
+} from './UserRepos.styles';
+import RepoMeta from '../repo-meta';
+import PrivacyBadge from '../misc/Privacy-badge';
+
+function UserRepos({ isOrg = false }) {
+  const [repos, setRepos] = useState<Repo[]>();
+  const [loading, setLoading] = useState(true);
+  const params = useParams();
+
+  const GITHUB_URL = isOrg ? ORG_REPO_LIST(params.username!) : ''; // The empty string can be replaced with URL to get all User Repos.
```

Comment:
I don't like the non-null assertion operator in TypeScript. It's usually hiding a code smell. I also don't understand the empty string thing and why we're doing that here.

Reply:
Alright. This make sense